src/instance.ts

```diff
@@ -193,6 +249,264 @@ class Instance extends common.GrpcServiceObject {
     this.databases_ = new Map<string, Database>();
   }
 
+  /**
+   * Get a reference to a Backup object.
+   *
+   * @throws {Error} If any parameter is not provided.
+   *
+   * @param {string} backupId The name of the backup.
+   * @return {Backup} A Backup object.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const backup = instance.backup('my-backup');
+   */
+  backup(backupId: string): Backup {
+    if (!backupId) {
+      throw new Error('A backup ID is required to create a Backup.');
+    }
+
+    return new Backup(this, backupId);
+  }
+
+  getBackups(options?: GetBackupsOptions): Promise<GetBackupsResponse>;
+  getBackups(callback: GetBackupsCallback): void;
+  getBackups(query: GetBackupsOptions, callback: GetBackupsCallback): void;
+  /**
+   * Query object for listing backups.
+   *
+   * @typedef {object} GetBackupsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetBackupsResponse
+   * @property {Backup[]} 0 Array of {@link Backup} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List backups on the instance.
+   *
+   * Both completed and in-progress backups are listed if no filter is supplied.
+   *
+   * @see {@link #backup}
+   *
+   * @param {GetBackupsOptions} [options] Contains query object for listing
```

Comment:
This description is for `options.gaxOptions`. `options` is: https://github.com/prunge-helix/nodejs-spanner/blob/prunge/backups-samples-2020-premerge/src/instance.ts#L280

Reply:
Makes sense. Here and below, I separated out the options parameter to [options] and [options.gaxOptions].